Palmer_McKay/detail.js

```diff
@@ -0,0 +1,15 @@
+console.log(DMP_CONTEXT.get())
+$(function(context) {
+    return function() {
+        // Sets main image by hovering over thumnails
+        $(".thumnail-img").mouseover(function(){
```

Comment:
thumnail?

Reply:
Perhaps I could find a better name for the smaller product images to the left of the big product image.